setup.cfg

```
@@ -49,6 +49,7 @@ install_requires =
     graphviz
     matplotlib
     pandas < 1.1
+    shap @ git+https://github.com/slundberg/shap.git#egg=shap
```

[minor] I believe in this format the egg is not required:
```suggestion
    shap @ git+https://github.com/slundberg/shap.git
```